File: tools/python-execute-eval/attempts/claude-opus-4-6/notes-crud.py
async def main(params):
    for op in params["ops"]:
        if op["op"] == "write":
            await kody.call("notes.write", {"id": op["id"], "text": op["text"]})
        elif op["op"] == "remove":
            await kody.call("notes.remove", {"id": op["id"]})
    return await kody.call("notes.list")

File: tools/python-execute-eval/results/grok-4.7/notes-crud.mjs
import { kody } from 'kody:runtime'

export default async function main(params) {
	for (const op of params.ops) {
		if (op.op === 'write') {
			await kody.call('notes.write', { id: op.id, text: op.text })
		} else if (op.op === 'remove') {
			await kody.call('notes.remove', { id: op.id })
		}
	}
	return kody.call('notes.list', {})
}
